theia-extensions/viewer-prototype/src/common/trace-viewer-environment.ts

```diff
@@ -33,10 +33,8 @@ export class TraceViewerEnvironment {
 
     protected _traceServerUrl: string | undefined;
     async getTraceServerUrl(): Promise<string> {
-        if (!this._traceServerUrl) {
```

Comment:
There's no reason _traceServerUrl would already be set by something and that we would want to keep this value instead of using "TRACE_SERVER_URL"?